modP2pImpl/src/org/aion/p2p/impl/one/msg/Hello.java

```diff
@@ -49,12 +49,12 @@ public String getMsg(){
         return this.msg;
     }
 
-    public static Hello decode(final byte[] _bytes) {
-        String msg = "";
+    public static Hello decode(final byte[] _bytes) throws UnsupportedEncodingException {
+        String msg;
         try{
             msg = new String(_bytes, "UTF-8");
         } catch (UnsupportedEncodingException e) {
-
+            throw e;
```

Comment:
is there a need for the `try+catch` here?

Reply:
updated in https://github.com/aionnetwork/aion/pull/588/commits/461db179a715f3b3dab7b722bad99142f7b933f7